### ISSUE TITLE
Wrong implementation on remove method of Lawnchair-sqliteplugin 

### ISSUE THREAD

**Issue by ste4net** [brody2consult/cordova-sqlite-storage#464]
Hello,
Reading Lawnchair api documentation i saw that remove method accept also key array.
Here is the quick modification i have done to Lawnchair plugin to work with key array.
Thank you

```
remove: function (keyOrObj, cb) {
            var that = this
            , key = typeof keyOrObj === 'string' ? keyOrObj : (Array.isArray(keyOrObj) ? null : keyOrObj.key)
            , del = "DELETE FROM " + this.name + (Array.isArray(keyOrObj) ? " WHERE id IN ('" + keyOrObj.join("','") + "')" : " WHERE id = ?")
            , win = function () { if (cb) that.lambda(cb).call(that) }

            this.db.transaction(function (t) {
                if (key == null)
                    t.executeSql(del, [], win, fail);
                else
                    t.executeSql(del, [key], win, fail);
            });

            return this;
        }
```

**Comment by brody2consult** [comment]
Thanks for your interest. To be honest I have not look at the Lawnchair adapter for the past several years and may move it to a separate project.

It would be great if you could issue a pull request, along with a statement that you have tested it to a reasonable extent. It would make maintenance a little easier and also be sure to give you the credit for the fix.

**Comment by brody2consult** [comment]
Included #465, thanks!
